activemodel/lib/active_model/type/collection.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module ActiveModel
+  module Type
+    # Attribute type for a collection of values. It is registered under
+    # the +:collection+ key. +:element_type+ option is used to specify elements type
+    #
+    #   class User
+    #     include ActiveModel::Attributes
+    #
+    #     attribute :lucky_numbers, :collection, element_type: :integer
+    #   end
+    #
+    #   user = User.new(lucky_numbers: [1, 2, 3])
+    #   user.lucky_numbers # => [1, 2, 3]
+    #
+    # Value is wrapped into an Array if not an Array already
+    #
+    #   User.new(lucky_numbers: 1).lucky_numbers # => [1]
+    #
+    # Collection elements are coerced by their +:element_type+ type
+    #
+    #   User.new(lucky_numbers: ["1"]).lucky_numbers # => [1]
+    #
+    class Collection < Value
+      def initialize(**args)
+        @element_type = args.delete(:element_type)
```

Comment:
I read through  https://github.com/rails/rails/pull/44324 and didn't see any discussion about or mention of alternative names for the `:element_type` keyword. Were other, shorter names considered as alternatives to `:element_type` option, like `:of` or `:type`? 

```ruby
attribute :values, :collection, of: :integer

# or
attribute :values, :collection, type: :integer
```